src/index.spec.ts

```diff
@@ -463,6 +473,82 @@ describe('Try batch operations', () => {
      });
 });
 
+describe('verify proofs existence and non-existence', () => {
+  // Keys to create branch, extension and leaf nodes
+  const k1 = Buffer.from('8a40bfaa73256b60764c1bf40675a99083efb075', 'hex');
```

Comment:
This test is sort of complex. It's good, but it would be nice to have a single unit test that tests for a non-existent node attached to one of each type of node.